src/lib.rs

```diff
@@ -26,6 +29,33 @@ pub struct rustls_client_session {
     _private: [u8; 0],
 }
 
+// Keep in sync with Cargo.toml.
+const RUSTLS_CRATE_VERSION: &str = "0.19.0";
+
+/// Write the version of the crustls C bindings and rustls itself into the
+/// provided buffer, up to a max of `len` bytes. Output is UTF-8 encoded
+/// and NUL terminated. Returns the number of bytes written before the NUL.
```

Comment:
I'd suggest not doing NUL terminated here.

Reply:
`curl` expects NUL-terminated strings when calling `Curl_ssl_version`, which will call this function: https://github.com/curl/curl/blob/master/lib/version.c#L165.

If I don't do NUL termination on this side, I'd just have to add it on the C side. I'd rather do it on the Rust side.